transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpProtocolCompatibilityFilter.java

```diff
@@ -425,6 +417,19 @@ else if (!PROTOCOL_HTTPXE_1_1.equals(nextProtocol)) {
             }
         }
 
+        // Use .ksn for x-sequence-no header for wse/1.0, httpxe/1.1 cases
+        // other cases, don't do anything as some services (for e.g http.proxy) want to pass through
+        String protocol = httpRequest.getHeader(HEADER_X_NEXT_PROTOCOL);
```

Comment:
Do you want to get this from the sessions resource address `address.getOption(NEXT_PROTOCOL)`?  I would imagine the http proxy would still treat requests that have that header value with the special rewrite behavior, correct?